src/style.css

```diff
@@ -1 +1,51 @@
 /* add styles here */
+
+.page {
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+}
```

Comment:
According to the checklist, avoid using tag names for styling, with the exception of `html` and `body`. Here, the `.page` class seems to be intended as a global style that could be applied directly to the `body` tag, which is an exception to this rule. If `.page` is indeed meant for the `body`, consider applying these styles directly to the `body` tag.